frontstage/views/surveys/upload_survey.py

```diff
@@ -34,37 +34,50 @@ def upload_survey(session):
     party_controller.is_respondent_enrolled(party_id, business_party_id, survey_short_name)
 
     # Get the uploaded file
-    upload_file = request.files['file']
-    upload_filename = upload_file.filename
+
+    # The variable name has '_raw' so that it can be called in the file size checking function,
```

Comment:
Comment probably isn't worth it.  2-3 lines later is a variable called `upload_file` so it's pretty clear why it's called this without the comment